netzsch/.htaccess

```diff
@@ -0,0 +1,29 @@
+RewriteEngine On
+
+# Rule 1: Redirect base to index
+RewriteRule ^$ https://heu-batmachine.github.io/netzsch-public-resources/index.html [R=301,L]
+
+# Rule 2: Redirect term IRI to human readable documentation
+RewriteRule ^(public/)?ontology(|#(.*))$ https://heu-batmachine.github.io/netzsch-public-resources/pages/netzsch.html$2 [R=301,L]
```

Comment:
User agents are not likely to send hash fragments, if that's what this is trying to match.  Browsers will likely do what you expect and add it to a redirected URL.